2016/03.md

```diff
@@ -63,10 +63,10 @@
   1. New proposals for Stage 1
     1. ~~[Private Fields](https://zenparsing.github.io/es-private-fields/) (Kevin Smith)~~
     1. [Weak References](https://github.com/tc39/proposal-weakrefs) (Dean Tribble)
-    1. ~~[Draft Proposed Frozen Realm API](https://github.com/FUDCo/frozen-realms) (Chip Morningstar, Mark Miller)~~
+    1. ~~[Draft Proposed Frozen Realm API](https://github.com/FUDCo/frozen-realms) (Chip Morningstar, Mark Miller) [Slides](https://github.com/FUDCo/frozen-realms/blob/master/FrozenRealmsPresentation-2016-03-30.pdf)~~
     1. [Lifting TemplateCharacter Restrictions](https://github.com/disnet/template-literal-revision) (Tim Disney)
     1. [Agents (spun out from Shared Memory and Atomics) (tentative)](https://axis-of-eval.org/shmem/agents-formatted.html) [Slides](https://github.com/tc39/ecmascript_sharedmem/raw/master/tc39/agents-mar-2016.odp) (Lars Hansen)
-    1. [Map.prototype.toJSON/Set.prototype.toJSON](https://github.com/DavidBruant/Map-Set.prototype.toJSON) (Jordan Harband, David Bruant)
+    1. ~~[Map.prototype.toJSON/Set.prototype.toJSON](https://github.com/DavidBruant/Map-Set.prototype.toJSON) (Jordan Harband, David Bruant)~~
```

Comment:
this discussion isn't actually finished yet - i was interrupted when brendan arrived :-)